primitives/xcm-calls/src/staking.rs

```diff
@@ -228,10 +229,9 @@ pub struct StakingWeights {
     pub withdraw_unbonded: Weight,
 }
 
-impl StakingWeights {
+impl Default for StakingWeights {
```

Comment:
hesitant against making this `Default` over `polkadot`, because we'll also need to add support for kusama/westend etc. so keeping dedicated function for each runtime is probably better. 
Also this should be moved to dedicated runtime mod later once we ad a pint runtime for kusama/polkadot

Reply:
What about 

```rust
trait XcmCallWeights {
    fn polkadot() -> Self;
}

impl XcmCallWeights for StakingWeights {
  fn polkadot() -> Self {
      Default::default()
  }
}
```

Reply:
this is reasonable, we can put this later in `runtime/common` and add `fn kusama()`
But I'm in favor of dropping `Default`, rather use it directly in `polkadot`.
naming suggestion: `XcmRuntimeCallWeights`?